draft-ietf-quic-recovery.md

```diff
@@ -1300,7 +1304,7 @@ largest_acked_packet\[kPacketNumberSpace]:
 : The largest packet number acknowledged in the packet number space so far.
 
 loss_time\[kPacketNumberSpace]:
-: The time at which the next packet in that packet number space will be
+: The earliest time at which an additional packet in that packet number space will be
```

Comment:
What was wrong with the original sentence?

Reply:
"next" could be ambiguous about applying to the time axis or the packet number axis.

Reply:
As packet numbers are used sequentially, those axis should be effectively equivalent.

Reply:
+1 to Marten's comment, reverting most of it, but changing will to can, since timers can wake up late.